modules/sensors/max11609.h

```diff
@@ -0,0 +1,33 @@
+#ifndef MAX11609_
+#define MAX11609_
+
+/**
+ * @brief Rate at which the MAX11609 sensors are read (in ms)
+ */
+#define MAX11609_UPDATE_RATE       2000
```

Comment:
Change this delay to 200 ms. Technically this is not an 'update rate' but an update period (this should be fixed in other sensor reading tasks in the future).